policy/reviewer/reviewer.go

```diff
@@ -118,35 +118,53 @@ func selectAdmins(prctx pull.Context) ([]string, error) {
 			if err != nil {
 				return nil, errors.Wrapf(err, "Unable to get list of members for %s", team)
 			}
-			adminUsers = append(adminUsers, admins...)
+			adminTeams[fullTeamName] = admins
 		}
 	}
 
-	return adminUsers, nil
+	return adminTeams, nil
 }
 
-func SelectReviewers(ctx context.Context, prctx pull.Context, result common.Result, r *rand.Rand) ([]string, error) {
+func getPossibleReviewers(prctx pull.Context, users []string, collaboratorsToConsider map[string]string) []string {
+	var possibleReviewers []string
+	for _, u := range users {
+		// Remove the author and any users who aren't collaborators
+		// since github will fail to assign _anyone_ if the request contains one of these
+		_, ok := collaboratorsToConsider[u]
+		if u != prctx.Author() && ok {
+			possibleReviewers = append(possibleReviewers, u)
+		}
+	}
+	return possibleReviewers
+}
+
+func SelectReviewers(ctx context.Context, prctx pull.Context, result common.Result, r *rand.Rand) ([]string, []string, error) {
 	var usersToRequest []string
+	var teamsToRequest []string
```

Comment:
I think these need to be initialized as `[]string{}` so that the function always returns non-nil slices when using only one type of assignment. The GitHub API / `go-github` requires passing non-nil slices in the `github.ReviewersRequest` struct.